files/zh-cn/web/html/element/input/image/index.md

```diff
@@ -7,83 +7,323 @@ slug: Web/HTML/Element/Input/image
 
+{{EmbedInteractiveExample("pages/tabbed/input-image.html", "tabbed-standard")}}
+
 <table class="properties">
- <tbody>
-  <tr>
-   <th scope="row">Permitted content</th>
-   <td>None, it is an {{Glossary("empty element")}}.</td>
-  </tr>
-  <tr>
-   <th scope="row">Tag omission</th>
-   <td>Must have a start tag and must not have an end tag.</td>
-  </tr>
-  <tr>
-   <th scope="row">Permitted parent elements</th>
-   <td>Any element that accepts <a href="/zh-CN/docs/HTML/Content_categories#Phrasing_content">phrasing content</a>.</td>
-  </tr>
-  <tr>
-   <th scope="row">DOM interface</th>
-   <td>{{domxref("HTMLInputElement")}}</td>
-  </tr>
- </tbody>
+  <tbody>
+    <tr>
+      <td><strong><a href="#值">值</a></strong></td>
+      <td>无 - <code>value</code> 属性不应该被指定。</td>
+    </tr>
+    <tr>
+      <td><strong>事件</strong></td>
+      <td>无</td>
+    </tr>
+    <tr>
+      <td><strong>支持的公共属性</strong></td>
+      <td>
+        <a href="/zh-CN/docs/Web/HTML/Element/input#alt"><code>alt</code></a>,
+        <a href="/zh-CN/docs/Web/HTML/Element/input#src"><code>src</code></a>,
+        <a href="/zh-CN/docs/Web/HTML/Element/input#width"><code>width</code></a>,
+        <a href="/zh-CN/docs/Web/HTML/Element/input#height"><code>height</code></a>,
+        <a href="/zh-CN/docs/Web/HTML/Element/input#formaction"><code>formaction</code></a>,
+        <a href="/zh-CN/docs/Web/HTML/Element/input#formenctype"><code>formenctype</code></a>,
+        <a href="/zh-CN/docs/Web/HTML/Element/input#formmethod"><code>formmethod</code></a>,
+        <a href="/zh-CN/docs/Web/HTML/Element/input#formmethod"><code>formnovalidate</code></a>,
+        <a href="/zh-CN/docs/Web/HTML/Element/input#formtarget"><code>formtarget</code></a>
+      </td>
+    </tr>
+    <tr>
+      <td><strong>IDL 属性</strong></td>
+      <td>无</td>
+    </tr>
+    <tr>
+      <td><strong>DOM 接口</strong></td>
+      <td><p>{{domxref("HTMLInputElement")}}</p></td>
+    </tr>
+    <tr>
+      <td><strong>方法</strong></td>
+      <td>无</td>
+    </tr>
+  </tbody>
 </table>
 
-## Attributes
+## 值
 
-This element includes the [global attributes](/zh-CN/docs/HTML/Global_attributes).
+`<input type="image">` 元素不接受 `value` 属性。要显示的图像的路径在 `src` 属性中指定。
 
-{{htmlattrdef("type")}}
+## 额外属性
 
-- {{htmlattrdef("formaction")}}
-  - : The URI of a program that processes the information submitted by the input element, here image if specified, it overrides the {{htmlattrxref("action","form")}} attribute of the element's form owner.
+除了所有 {{HTMLElement("input")}} 元素的共有属性外，`image` 按钮还支持下列属性。
+
+### alt
+
+`alt` 属性提供一个备用字符串，在图像无法加载的情况下作为按钮的标注（由于错误，一个不能或者被设置为不能显示图像的 {{Glossary("user agent")}}，或用户正在使用屏幕阅读设备）。如果它被提供，它必须为非空字符串，且适合作为按钮的标签。
+
+例如，如果你有一个按钮，显示一个带有图标和/或图像文本“现在登录”的图片，您也应该设置 `alt` 属性为类似于 `现在登录` 的东西。
+
+> **注意：** 虽然 `alt` 属性在技术上是可选的，但是您总应该包含它，以最大限度地提高您的内容的可用性。在功能上，`<input type="image">` 元素的 `alt` 属性与 {{HTMLElement("img")}} 元素的 {{htmlattrxref("alt", "img")}} 属性相类似。
```

Comment:
This is a special heighlight block in mdn, see: https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#%E5%A4%87%E6%B3%A8%E3%80%81%E8%AD%A6%E5%91%8A%E5%92%8C%E6%A0%87%E6%B3%A8

```suggestion
> **备注：** 虽然 `alt` 属性在技术上是可选的，但是您总应该包含它，以最大限度地提高您的内容的可用性。在功能上，`<input type="image">` 元素的 `alt` 属性与 {{HTMLElement("img")}} 元素的 {{htmlattrxref("alt", "img")}} 属性相类似。
```